core/templates/pair.h

```diff
@@ -74,7 +74,13 @@ struct KeyValue {
 	const K key;
 	V value;
 
-	void operator=(const KeyValue &p_kv) = delete;
+	void operator=(const KeyValue &p_kv) {
+		*this = p_kv;
+	}
+	_FORCE_INLINE_ KeyValue() :
+			key(K()),
+			value(V()) {
+	}
```

Comment:
I had to make this change to make `KeyValue` work with `List` but not sure this is the right approach.